packages/hoppscotch-sh-admin/src/pages/users/invited.vue

```diff
@@ -16,64 +16,36 @@
           <div v-if="fetching" class="flex justify-center">
             <HoppSmartSpinner />
           </div>
-          <div v-else-if="error || invitedUsers === undefined">
+          <div
+            v-else-if="
+              error || invitedUsers === undefined || invitedUsers.length === 0
+            "
+          >
             <p class="text-xl">{{ t('users.no_invite') }}</p>
```

Comment:
Couldn't find a locale entry corresponding to `users.no_invite`. Could you confirm if it's the intended usage? Also, shouldn't the case of error be handled separately with a different message?

Reply:
The entry is present in en.json file in sh-admin repo. I have now created a separate check for just the error case.